src/FindTranslocations.cpp

```diff
@@ -184,59 +66,71 @@ int main(int argc, char *argv[]) {
 	uint32_t minimumSupportingPairs = 10;
 	int min_insert				    = 100;      // min insert size
 	int max_insert				    = 1000000;  // max insert size
-	string outputFileHeader         = "output"; // default output name
 	int minimum_mapping_quality     = 20;
 	float coverage;
 	float meanInsert;
 	float insertStd;
 	string roi;
-
+	string indexFile="";
+	string outputFileHeader ="";
```

Comment:
yep, connected to previous comments this will allow to take away all these variables and put them in a object. As this Option object needs to be instantiated only once it might be worthwhile to implement it as a [Singleton](http://stackoverflow.com/questions/1008019/c-singleton-design-pattern).